frontend/src/views/SqlEditor/TabListContainer.vue

```diff
@@ -335,6 +367,16 @@ onMounted(async () => {
     recalculateScrollWidth();
   }
 });
+
+onMounted(() => {
+  window.onbeforeunload = () => {
+    return "false";
+  };
+});
+
+onUnmounted(() => {
+  window.onbeforeunload = null;
+});
```

Comment:
Do not clear about these block, can you add some comment.

Reply:
@xiaoluoboding use browser confirm to confrim if close the tab. 
<img width="408" alt="image" src="https://user-images.githubusercontent.com/41265413/156361390-37740e07-f1ed-4893-8dc5-41a169826795.png">